package-lock.json

```diff
@@ -1,8 +1,3074 @@
 {
```

Comment:
Whoa!   This `package-lock.json` should be deleted.  I don't know how it got here.

Reply:
This package actually has runtime, for the tests. For `yarn test` to work requires an install, which will always generate a lockfile.

I'll merge as is but happy to come back and solve the package lock issue if necessary.